packages/eslint-plugin-pf-codemods/lib/rules/v5/table-warn-actionsColumn.js

```diff
@@ -1,18 +1,35 @@
+const { getPackageImports } = require('../../helpers');
+
 // https://github.com/patternfly/patternfly-react/pull/8629
 module.exports = {
   create: function (context) {
     return {
       ImportDeclaration(node) {
-        const TableImport = node.specifiers.find(
+        const coreImports = getPackageImports(context, '@patternfly/react-core');
+        const deprecatedImports = getPackageImports(context, '@patternfly/react-core/deprecated');
+        const tableImports = getPackageImports(context, '@patternfly/react-table');
+        const deprecatedTableImports = getPackageImports(context, '@patternfly/react-table/deprecated');
+        const imports = [...coreImports, ...tableImports, ...deprecatedImports, ...deprecatedTableImports];
+
+        const tableImport = imports.find(
           (specifier) =>
-            node.source.value === "@patternfly/react-table" &&
             (specifier.imported.name === "Table" || specifier.imported.name === "TableComposable")
         );
 
-        if (TableImport) {
+        const dropdownToggleImport = imports.find(
+          (specifier) =>
+            (specifier.imported.name === "DropdownToggle")
+        );
+
+        const actionsColumnImport = imports.find(
+          (specifier) =>
+            (specifier.imported.name === "ActionsColumn")
+        );
+
+        if (tableImport && (dropdownToggleImport || actionsColumnImport)) {
           context.report({
             node,
-            message: "The ActionsColumn now uses the Next version of Dropdown. The action toggle should now pass a MenuToggle rather than a DropdownToggle, and direction and position properties are now passed under the new popperProps property.",
+            message: "The ActionsColumn now uses the Next version of Dropdown. The action toggle should now pass a MenuToggle rather than a DropdownToggle, and direction and position properties are now passed under the ActionsColumn new popperProps property.",
```

Comment:
Latest update should work.. I think there's pros and cons to targeting only the deprecated path for DropdownToggle vs both react-core and deprecated like the latest update does.

Updating the warning verbiage may help:

```suggestion
            message: "The ActionsColumn within Table now uses our new implementation of Dropdown. The action toggle should now pass a MenuToggle rather than the deprecated DropdownToggle, and direction and position properties are now passed under the ActionsColumn new popperProps property.",
```

Maybe make the "within Table" dynamic so that it uses whatever name is found from tableImport, wdyt?

Could we also update the README for this codemod so that the opening sentence states, "...has been updated to use our new implementation of Dropdown"